Tests/SwiftParserTest/translated/AvailabilityQueryUnavailabilityTests.swift

```diff
@@ -473,4 +477,16 @@ final class AvailabilityQueryUnavailabilityTests: XCTestCase {
       ]
     )
   }
+
+  func testAvailabilityQueryUnavailability34d() {
+    assertParse(
+      """
+      if #available(*) 1️⃣== {
+      }
+      """,
+      diagnostics: [
+        DiagnosticSpec(locationMarker: "1️⃣", message: "unexpected code '==' in 'if' statement")
```

Comment:
Maybe we should add a fixit if we start to require it in #1467 

Same for fixedSource below

Reply:
Thanks for the reminder